openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -136,7 +136,6 @@ def tearDown(self):
         remove_course_updates(self.user, self.course)
         super(TestCourseHomePage, self).tearDown()
 
-    @override_waffle_flag(UNIFIED_COURSE_TAB_FLAG, active=True)
```

Comment:
Just curious: These changes were not needed, right?  You were just cleaning up since they were no longer necessary?

Reply:
yup